classes.js

```diff
@@ -0,0 +1,78 @@
+// # Class Exercises
+
+// 1.
+//   * Create a Human class that takes in a name and age.
+//   * Add the function `ageOneYear` that ages the human.
+//   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
+//   * Create an instance of the Human class.
+//     * console log your humans age
+//     * call ageOneYear on your human
+//     * console log their age again.
+//     * call eating on your human.
+
+class Human {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+
+  ageOneYear() {
+    return this.age + 1
+  }
+
```

Comment:
The ageOneYear function isn't quite correct. The function should add one to the value of the particular Human age. This function returns the age plus 1 but it doesn't change the value. You want this.age++, this.age += 1 or this.age = this.age + 1. Let me know if this isn't clear.